### ISSUE TITLE
Jabra Unexpected endcall

### ISSUE THREAD

**Issue by nnew2** [gnaudio/jabra-browser-integration#58]
Hello,

We're receiving unexpected endcall events from what seems to be the Chromehost

[chrome_chromehost.txt](https://github.com/gnaudio/jabra-browser-integration/files/4317883/chrome_chromehost.txt)
[chrome_console.txt](https://github.com/gnaudio/jabra-browser-integration/files/4317884/chrome_console.txt)

I attached files from the Chrome extension background script and the chromehost logs.

Remarks:
1) In the chrome_chromehost.txt the issue occurs at 2020-03-11 12:09:57.088
2) This is not an isolated incident it happens widespread across all agents and multiple times in an hour.
3) Modell: Jabra BIZ 2400 II
4) Windows 10
5) The agent is nowhere near to touch the headset controller.

We're running Chromehost 2.0.1-2 and api version 2.0

**Comment by extmchristensen** [comment]
Can you test with the v3.0 beta - it has upgraded the embedded C-SDK which has many bugfixes for call control.

**Comment by nnew2** [comment]
We received words from your Developer zone that we could expect an update in mid April. I think we are OK to wait for this release to avoid running beta in production.

**Comment by extmchristensen** [comment]
Yes - don't run beta's in production but if you can test on a test system with our new beta that would be great.
